templates/macros/format_date.html

```diff
@@ -2,6 +2,8 @@
 
 {# Set locale #}
 {% set date_locale = macros_translate::translate(key="date_locale", default="en_GB", language_strings=language_strings) %}
+{% set long_date_format = macros_translate::translate(key="long_date_format", default="en_GB", language_strings=language_strings) %}
```

Comment:
🛠️
short format `default` should be `"%-d %b %Y"`
long format `default` should be `"%d %b %Y"`

Reply:
The reason is the same as the previous one